### PR TITLE
Upgraded to Jekyll 3.8

### PR DESCRIPTION
**Pull request by Kyrremann**
Also removed pygments.rb as it is not supported on Github pages.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,41 +1,38 @@
-# Title of your blog; shows in the header
-name: "hikari"
+name: "kyrremann"
```
Comment (ssorallen) [inline _config.yml:1]: This PR contains a lot of change that look like they're personal and for your site. Can you put up a PR containing only the changes necessary to upgrade Jekyll?

**Review by ssorallen — CHANGES_REQUESTED** [review]
Thanks a lot for this. Can you constrain your changes to just the Jekyll upgrade? Looks like this PR caught a lot of changes for your personal site.

**Comment by Kyrremann** [comment]
Haha, that's a mistake from my side. I'll create a new pull request tonight.

**Comment by Kyrremann** [comment]
Created a new PR https://github.com/mx3m/hikari-for-jekyll/pull/48

If you are interested in the pygments fix, I can make a separated PR for that.

**Pull request closed without merging**